src/main/resources/patcher.mixins.json

```diff
@@ -7,7 +7,6 @@
     "conformVisibility": true
   },
   "client": [
-    "accessors.AbstractClientPlayerAccessor",
```

Comment:
Generally avoid unnecessary changes like these & the RegionRenderCacheMixin needs to be in this file.